docs/en/observability/view-observability-alerts.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[view-observability-alerts]]
+= View alerts
+
+experimental[]
+
+****
+Given the experimental status of this feature, if you need to disable and remove the Alerts page,
+you can do so using the following configuration settings:
+
+`xpack.ruleRegistry.write.enabled: 'false'`
+`xpack.observability.unsafe.cases.enabled: 'false'`
+`xpack.observability.unsafe.alertingExperience.enabled: 'false'`
+
+If you have a self-managed {stack}, edit the settings in your `kibana.yml` file.
+
+If you are using our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+edit the {kib} user settings:
+
+. Select your deployment on the home page, and from your deployment menu,
+go to the *Edit* page.
+. In the *{kib}* section, click *Edit user settings*, and add the settings if they are not already present.
+. Click *Back*, and then click *Save*. The changes are
+automatically appended to the `kibana.yml` configuration file for your instance.
+
+****
+
+The Alerts page lists all the alerts that have met a condition defined by a rule you created using the Logs,
+Metrics, Uptime, or APM apps.
+
+After alerts have been triggered, you can monitor their activity to verify they are functioning correctly.
+In addition, you can filter alerts, update their workflow status, and troubleshoot each alert in their
+respective app.
+
+You can also add alerts to <<create-cases,Cases>> to open and track potential infrastructure issues.
+
+NOTE: You can centrally manage rules from the
+{kibana-ref}/create-and-manage-rules.html[{kib} Management UI] that provides a
+set of built-in {kibana-ref}/rule-types.html[rule types] and
+{kibana-ref}/action-types.html[connectors] for you to use. Click *Manage Rules*
+
+[role="screenshot"]
+image::images/alerts-page.png[Alerts page]
+
+[discrete]
+[[filter-observability-alerts]]
+== Filter alerts
+
+To help you get started with your analysis faster, use the KQL bar to create structured queries using
+{kibana-ref}/kuery-query.html[Kibana Query Language]. For example, `kibana.alert.status: "recovered"`.
+
+You can also use the time filter to define a specific date and time range. By default, this filter is set to search
+for the last 15 minutes.
+
+To view alerts that you have already taken action on, select either the *Acknowledged* or *Closed* tabs.
```

Comment:
As I mentioned above soon the Workflow status (Open/Acknowledged/Closed) will be replaced by the alert status (Active/Recovered) https://github.com/elastic/kibana/issues/110242. Keep that in mind for 8.0

Reply:
Thanks. Tracking with this [issue](https://github.com/elastic/observability-docs/issues/1260).